automation/jleechanorg_pr_automation/automation_utils.py

```
@@ -12,22 +12,22 @@
+from typing import Any, Dict, Optional, Sequence, Tuple
 
-try:
-    import keyring
-    KEYRING_AVAILABLE = True
-except ImportError:
-    KEYRING_AVAILABLE = False
+import keyring
+
+KEYRING_AVAILABLE = True
+
+from .utils import json_manager, setup_logging as shared_setup_logging
```

Ruff (0.14.10)

Module level import not at top of file

(E402)